snafu-derive/src/lib.rs

```diff
@@ -638,6 +640,15 @@ const ATTR_CRATE_ROOT: OnlyValidOn = OnlyValidOn {
     valid_on: "an enum or a struct",
 };
 
+const ATTR_TRANSPARENT: OnlyValidOn = OnlyValidOn {
```

Comment:
We should add compile-fail tests for the "special" new errors. The transparent + context error cases especially, as that's kind of a new pattern and not as easy as e.g. one more usage of `AtMostOne`.

(`cd compatibility-tests/compile-fail && cargo test`).

Reply:
Just added it the last commit